src/pages/LandingPage.svelte

```diff
@@ -0,0 +1,44 @@
+<script lang="ts">
+  import "../style/main.css";
+  // @ts-expect-error no declarations
+  import { initAll } from "govuk-frontend";
+  import {
+    AlphaBanner,
+    DefaultButton,
+  } from "govuk-svelte";
+  import { Header } from "lib/common";
+  import { onMount } from "svelte";
+
+  onMount(async () => {
+    // For govuk components. Must happen here.
+    initAll();
+  });
+
+  function goToSketcher() {
+    window.location.href = "/choose_area.html"

```

Comment:
Please test your changes on the deployed version too! The links on https://acteng.github.io/atip/landing-page/ are broken; don't use absolute links, so it'll work in a variety of deployed environments. Just remove the / here and below